src/checkbox/checkbox.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import style from './checkbox.module.css'
+
+const Checkbox = ({disabled, checked, onChange}) => {
+    return (
+        <div>
+            <input className={style.toggle}
+                   disabled={disabled}
+                   checked={checked}
+                   onChange={onChange}
+                   id='checkbox'
```

Comment:
ideally `id` should be dynamic. as in your `radio` component. if I render multiple checkboxes on the page then they will all have the same id with current implementation. this is not allowed by HTML standards